guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -344,15 +315,9 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
                         <ButtonLineComponent label="Save" onClick={() => {
                             this.save();
                         }} />
-                        {
-                            this.props.globalState.customSave &&
-                            <ButtonLineComponent label={this.props.globalState.customSave!.label} onClick={() => {

```

Comment:
Was this added somewhere else? or is it simply not needed?

Reply:
Yes this is not needed at the moment but will come back eventaully.